### ISSUE TITLE
Stack Exchange Open Source

### ISSUE THREAD

**Issue by vsoch** [sfosc/sfosc#90]
Just discovered this exchange in beta - https://opensource.stackexchange.com/ 

it's a good source for interesting conversation about licenses, open source, etc. Other than reading it, should / can we provide some links of good resources for users to find information and ask questions? If it's not appropriate for the web rendering, it could be added to the README or similar.

**Comment by Beanow** [comment]
Had another look at this today. Seems to have gotten a solid amount of interesting licensing/legal questions. Though I don't see much on subjects like governance, communities or sustainability just yet. So right now I think it could be a place we can cross-link on some occasions.

What I wonder about is, was this the intended scope for the exchange or just what got good reception now. Would a #sfosc tag fit there and be useful? If that's a thing, we could endorse that as a way to ask questions. However I'm still a bit concerned about having too many channels in the early days.

**Comment by vsoch** [comment]
I don't think sfosc would be a tag or category, but rather if any of the community here got involved answering questions there (and linked back to content on the site) it would help to grow the community and create awareness for the initiative.

**Comment by vsoch** [comment]
And it's definitely the case that (most) questions aren't a perfect match - I see this not being because they are off topic, but because these kinds of questions come at later stages of involvement with open source. I certainly wouldn't (and didn't) think about many of the ideas discussed here until I had been an "open source person" for quite a few years and started to experience first hand problems, and questions I didn't have answers to.

**Comment by Beanow** [comment]
Hmm I was thinking about off-topic more in the sense of, should we receive questions on SFOSC along the lines of: can I do X with license Y? We could refer them.

> I don't think sfosc would be a tag or category, but rather if any of the community here got involved answering questions there (and linked back to content on the site) it would help to grow the community and create awareness for the initiative.

I like the idea of finding places we can help and link back here in the process :]
I suppose we can use the search function to find specific questions we have resource on?

**Comment by vsoch** [comment]
Yes, I can offer to help to do this, likely in the evenings / weekend. LMK if you want to try it out.

**Comment by vsoch** [comment]
Here are some questions that members here might be able to provide good answers to (and link to SFOSC):

 - [Will open source licenses enable me to choose the business model I want](https://opensource.stackexchange.com/questions/468/will-open-source-licenses-enable-me-to-choose-the-businesses-model-i-want)
 - [What is the open chain project?](https://opensource.stackexchange.com/questions/7924/what-is-the-openchain-project) I thought this might fall in line with talking about business models (and if so, we should mention it somewhere too, seems important).
 - [Is there any research on whether the owner of a project making money from it has a significant effect on contributions](https://opensource.stackexchange.com/questions/8442/is-there-any-research-on-whether-the-owner-of-a-project-making-money-from-it-h) This is obviously hugely important for any open source project with a business model. I don't know the answer but I'd really like to :)

The sheer number of questions about licenses tells me that there is huge room for improvement in helping people to understand the implications of any choice. I hope that this Polyform project can start that change!
